src/components/main-page/Mentors/Mentors.tsx

```diff
@@ -9,18 +10,25 @@ export function Mentors() {
     <Section variant="blue" id="mentors">
       <Heading withAccent>{t("mentors.title")}</Heading>
       <div className={styles.cardContainer}>
-        <Card name="Some Name" company="Some Company" role="Some Role" />
-        <Card name="Some Name" company="Some Company" role="Some Role" />
-        <Card name="Some Name" company="Some Company" role="Some Role" />
-        <Card name="Some Name" company="Some Company" role="Some Role" />
-        <Card name="Some Name" company="Some Company" role="Some Role" />
-        <Card name="Some Name" company="Some Company" role="Some Role" />
-        <Card name="Some Name" company="Some Company" role="Some Role" />
-        <Card name="Some Name" company="Some Company" role="Some Role" />
-        <Card name="Some Name" company="Some Company" role="Some Role" />
-        <Card name="Some Name" company="Some Company" role="Some Role" />
-        <Card name="Some Name" company="Some Company" role="Some Role" />
-        <Card name="Some Name" company="Some Company" role="Some Role" />
+        {mentorsData.map((person) => (
+          <Card
+            name={person.name}
+            company={person.company}
+            role={
+              typeof person.role === "object"
+                ? person.role.reduce(
+                    (prev: string, next: string) =>
```

Comment:
It won't work if we'd have 3 elements, but it's ok if we have array of 2 or string - wouldn't it be better to explicitly use `${t("mentors.roles."+person.role[0])}/${t("mentors.roles." + person.role[1])}` to be more explicit?